Core/Core/AppEnvironment/AppEnvironment.swift

```diff
@@ -68,4 +72,23 @@ open class AppEnvironment {
         let useCase = LocalUseCase<Model>(scope: scope)
         return subscribe(useCase, callback)
     }
+
+    public func connectVideoPlayer(_ playerViewController: AVPlayerViewController) {
+        videoPlayerViewController = playerViewController
+        videoPlayer = playerViewController.player
+    }
+
+    public func backgroundVideoPlayer() {
+        videoPlayerViewController?.player = nil
+    }
+
+    public func reconnectVideoPlayer() {
+        videoPlayerViewController?.player = videoPlayer
+        videoPlayerViewController = nil
+    }
+
+    public func disconnectVideoPlayer() {
+        videoPlayerViewController = nil
+        videoPlayer = nil
+    }
```

Comment:
Is this the best place for this?  It seems easy and convenient to put here but AppEnvironment seems to be picking up all things at a good pace.  Would it be better to have some AudioVideo State singleton.  Just throwing that out there.  I'm not necessarily opposed to to it here either.